models/cf/models.go

```diff
@@ -73,8 +73,7 @@ type DesiredLRPLifecycleResponse struct {
 }
 
 type Lifecycle struct {
-	DockerLifecycle    *DockerLifecycle    `json:"docker_lifecycle"`
-	BuildpackLifecycle *BuildpackLifecycle `json:"buildpack_lifecycle"`
+	DockerLifecycle *DockerLifecycle `json:"docker_lifecycle"`
```

Comment:
Admittedly, now it does not make much sense to have a `Lifecycle` struct with only one possible lifecycle type struct inside. It would be much better if we take everything from `DockerLifecycle` and put it in `Lifecycle`. 

But changing this would require to also change the Cloud Controller code. Therefore, for the time being, I'm leaving it as it is.